cabal-install/main/Main.hs

```diff
@@ -304,7 +305,8 @@ mainWorker args = do
       , hiddenCmd  win32SelfUpgradeCommand win32SelfUpgradeAction
       , hiddenCmd  actAsSetupCommand actAsSetupAction
       , hiddenCmd  manpageCommand (manpageAction commandSpecs)
-
+      , hiddenCmd CmdShowBuildInfo.showBuildInfoCommand
```

Comment:
should it be a hidden command?

Reply:
Someone else needs to define this, I dont know enough of the process and the ramifications of such promises.